pkg/pool-stable-phantom/contracts/StablePoolProtocolFees.sol

```diff
@@ -96,19 +115,20 @@ abstract contract StablePoolProtocolFees is StablePoolStorage, StablePoolRates,
         // amplification are not translated into changes to the invariant. Since amplification factor changes are both
         // infrequent and slow, they should have little effect in the pool balances, making this a very good
         // approximation.
-        uint256 lastPostJoinExitAmp = _lastPostJoinExitAmp;
+
+        (uint256 lastJoinExitAmp, uint256 lastPostJoinExitInvariant) = getLastJoinExitData();
 
         uint256 swapFeeGrowthInvariant = StableMath._calculateInvariant(
-            lastPostJoinExitAmp,
+            lastJoinExitAmp,
             _getAdjustedBalances(balancesWithoutBpt, true) // Adjust all balances
         );
 
         uint256 totalNonExemptGrowthInvariant = StableMath._calculateInvariant(
-            lastPostJoinExitAmp,
+            lastJoinExitAmp,
             _getAdjustedBalances(balancesWithoutBpt, false) // Only adjust non-exempt balances
         );
 
-        uint256 totalGrowthInvariant = StableMath._calculateInvariant(lastPostJoinExitAmp, balancesWithoutBpt);
+        uint256 totalGrowthInvariant = StableMath._calculateInvariant(lastJoinExitAmp, balancesWithoutBpt);
```

Comment:
The balances are all getting renamed (registeredBalances, balances). Maybe wait until... and you merged it before I finished writing :) Anyway, update the names here too.